index.js

```diff
@@ -712,6 +712,28 @@ instance.prototype.actions = function() {
 				}
 			]
 		},
+		'set_volume' : {
+			label: 'Set Source Volume',
+			options: [
+				{
+					type: 'dropdown',
+					label: 'Source',
+					id: 'source',
+					default: '',
+					choices: self.sourcelist
+				},
+				{
+					type: 'number',
+					label: 'Volume in dB (-100 to 26) ',
+					id: 'volume',
+					default: null,
```

Comment:
why default `null`?

Reply:
Good question. My thinking here is because if you send null it won't change your current volume if you haven't entered a value quite yet. But, it you set to 0 it changes it to 0db. 

Not sure if that's a good reason, or what would be a good alternative

Reply:
I don't think that makes sense since its the only parameter and is the point of the action.  And you don't have any mitigation in the execution if the value is still `null` (though maybe OBS just ignores the action in that case).

Reply:
I agree with that. I changed the default to 0 and gave it a test, since that is what new sources start as in OBS.